src/envoy/http/jwt_auth/sample/APToken/key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
What are these key/cert file for. Please add a README if they are used to generate test data, and/or how to replace this file if need to.

Reply:
These key/cert files have been removed. I have added a guide of sending an example exchanged token to the jwt-authn filter and the Istio authn filter, and observing that the example backend echoes back the request when the authentication succeeds.